callstack/react-native-pager-view
# Fix indicator animation on setPage

# Summary

This pull request fixes indicator animation issue on _setPage_ method call on **iOS**. 

When calling _setPage_ method to page which was more than 1 index away for current one, indicator wasn't smoothly animating to new index.

Pull request also fixes incorrect offset value issue, which was causing indicator to "jump" a bit on page change.

Also fixes transition animation issue on **Paper** arch, which was causing to show transition animation from wrong page when navigating with _setPage_ method.

Fixes #636 

## Test Plan

1. open paper/fabric example app
2. open `Basic Example`
3. navigate to last page by pressing `Last` on navigation panel
4. indicator should animate correctly

## Compatibility

| OS      | Implemented |
| ------- | :---------: |
| iOS     |    ✅     |
| Android |    ❌     |

## Checklist
